packages/sources/alpine/test-payload.json

```diff
@@ -0,0 +1,3 @@
+{
+ "requests": []
```

Comment:
We recommend including some example queries here so you can verify the EA is working